integration-cli/daemon.go

```diff
@@ -268,6 +269,16 @@ func (d *Daemon) Kill() error {
 	return nil
 }
 
+// DumpStackAndQuit sends SIGQUIT to the daemon, which triggers it to dump its
+// stack to its log file and exit
+// This is used primarily for gathering debug information on test timeout
+func (d *Daemon) DumpStackAndQuit() {
+	if d.cmd == nil || d.cmd.Process == nil {
+		return
+	}
+	d.cmd.Process.Signal(os.Signal(syscall.SIGQUIT))
```

Comment:
On Windows, this needs to signal the mutex instead. See https://github.com/jhowardmsft/docker-signal/blob/master/docker-signal.go for an example of how to do this.